DesignSafe-CI/portal
# task/DES-2035: Advanced Search bugs

## Overview: ##
- The "Other" dropdown in Natural Hazard Type for field recon now searches for all hazard types NOT enumerated in the dropdown, instead of just the ones explicitly tagged as "Other".
- Visual and functional fixes for the Experiment Type dropdown.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2035](https://jira.tacc.utexas.edu/browse/DES-2035)

## Testing Steps: ##
1. Perform a search for Other type natural hazards in Field Recon, and check that you see more results in local dev than in staging.
